@xen-orchestra/lite/src/components/pool/dashboard/PoolDashboardStatus.vue

```diff
@@ -19,12 +19,12 @@
 </template>
 
 <script lang="ts" setup>
+import UiCardTitle from "@/components/ui/UiCardTitle.vue";
```

Comment:
No, the sort of the imports has been removed some times ago (because the whole configuration was leaking for an unknown reason, and it was not an really important feature)

Reply:
Ok, so what do we do now? /cc @MathieuRA @Rajaa-BARHTAOUI  Do we keep sorting them manually? That's what we did in XO 5 so far to avoid unnecessary diffs as much as possible.

Reply:
I think it would be preferable to continue to sort them to prevent unnecessary diffs, as you said.

On my side, I just checked my IDE (WebStorm) settings and found an option to do this automatically.

I suppose it should be possible to do this in VSCode too.

Reply:
Keep sorting them looks good to me.